scripts/generate_js_definitions.sh

```diff
@@ -46,3 +46,12 @@ fi
 # Generate the new packed tgz
 cd dist
 npm pack
+
```

Comment:
Next door to this file should also be updated: `scripts/update-js-dependencies.sh`

Reply:
Good catch. Thanks